modules/ROOT/pages/managing-users.adoc

```diff
@@ -50,6 +50,16 @@ Public key provided by the identity provider, which is used to sign the SAML ass
 +
 An arbitrary string value that identifies your Anypoint Platform organization. The typical value for this string is `<organizationDomain>.anypoint.mulesoft.com`.
 +
+* *Single Sign-On Initiation*
++
+Specify whether SSO can be initiated by the Anypoint Platform, your identity provider (for example, Okta), or both. 
+
+** The *Service Provider Only* option allows only the Anypoint Platform to initiate SSO. 
+** The *Identity Provider Only* option allows only your external identity provider to initiate SSO. 
+** The *Both* option allows either Anypoint Platform or your external identity provider to initiate SSO. 
++
+The default value for this setting for newly configured identity provider configurations is *Both (recommended)*. For existing identity provider configurations, the default setting is *Identity Provider Only*.
```

Comment:
We removed the `(recommended)` text from the UI, as it will mislead some users. We don't want to enforce anything on them. But still `Both` will be the default.

```suggestion
The default value for this setting for newly configured identity provider configurations is *Both*. For existing identity provider configurations, the default setting is *Identity Provider Only*.
```

Reply:
The default value for this setting for newly configured identity provider configurations is *Both*.